src/test/resources/data/FeedbackSessionsLogicTest.json

```diff
@@ -1203,7 +1203,7 @@
       "courseId": "idOfTypicalCourse1",
       "feedbackQuestionId": "1",
       "feedbackQuestionType": "TEXT",
-      "giver": "student4InCourse1@gmail.tmt",
+      "giver": "Team 1.1",
```

Comment:
The team name is `"Team 1.1</td></div>'\""` in the test data, fixing this and the issue with FeedbackSessionQuestionTypeTest.json should allow you to pass FeedbackSessionsLogicTest